README.md

```diff
@@ -38,8 +38,8 @@ $ sbt
 > exit
 $ cd target/sbt-test/default-*/scripted
 $ sbt
-$ jetty:start
-$ browse # starts browser for you, or manually open http://localhost:8080 to verify
+> jetty:start
+-> executes browser, and opens `http://localhost:8080` to verify
```

Comment:
You can hit `browse` after `jetty:start` -> `Enter`. This is my result after `g8Test`:
```bash
$ cd target/sbt-test/scalatra-sbt-g8/scripted/
$ sbt
...
> jetty:start
...
[success] Total time: 8 s, completed 2017/06/05 20:48:04
...
> browse
[info] Launching browser.
[success] Total time: 0 s, completed 2017/06/05 20:48:16
```
Then my browser was launched.